config/kubernetes/sig-api-machinery/teams.yaml

```
@@ -16,6 +16,22 @@ teams:
     - lavalamp
     - liggitt
     privacy: closed
+  sig-api-machinery-members:
+    description: Members of sig-api-machinery
+    members:
+    - deads2k
```

```suggestion
    - alexzielenski
    - deads2k
```

deleted my other inline comment. I see the list is sorted